teps/0079-tekton-catalog-support-tiers.md

```diff
@@ -517,7 +523,96 @@ TODO
 
 ### Image Scanning for Common Vulnerabilities and Exposures (CVEs)
 
-TODO
+#### CVEs Scanning Tool
+
+[Trivy](https://github.com/aquasecurity/trivy) is an open source, simple and comprehensive vulnerability scanner for container images and other artifacts, the advantages of which include but not limited to:
+
+1. **Comprehensive Coverage**: Trivy supports comprehensive vulnerability detection against a wide variety of OS Packages and application dependencies for 8 languages including Go. In addition to container image scanning, Trivy is also able to detect security issues targeting filesystem, git repository, kubernetes cluster and resources.
+2. **High Accuracy**: Trivy provides highly accurate security reports, especially Alpine Linux and RHEL/CentOS.
+3. **Performance & Scalability**: Trivy is fast and scalable thanks to the underlying static analysis technique. The first scan finishes within 10s of submission (depending on the network).
+4. **Easiness**: Trivy can be installed easily, executed both in standalone & client/server mode, and no prerequisite (DB, system library, env requirement...) is required.
+
+A comparison between Trivy and other CVEs scanning tools is available [here](https://aquasecurity.github.io/trivy/v0.17.2/comparison/). Given the above advantages, Trivy will be selected as the vulnerability scanning tool for Tekton Catalogs.
+
+#### Scanning Policy & Surface Vulnerability Report
+
+The Artifact Hub [Sanner Service](https://artifacthub.io/docs/topics/security_report/) uses Trivy as the underlying CVEs scanning tool, periodically running container image security reports for Helm Charts, OLM operators and 5 other kind of resource surfaced by the Artifact Hub. We propose to integrate Tekton Catalogs to the Artifact Hub [Sanner Service](https://artifacthub.io/docs/topics/security_report/) to run and surface Trivy security reports in the Artifact Hub.
+
+In the first iteration of the TEP, we will use the current scanning policy defined the Artifact Hub:
+1. The scanner runs twice an hour and scans packages’ versions that haven’t been scanned yet.
+2. The latest package version available is scanned daily.
+3. The previous versions are scanned weekly. 
+4. The Versions released more than one year ago won’t be scanned anymore.
+
+The container images must be stored in the registries that are publicly available. The security report will be displayed on the Artifact Hub UI for each version of the packages.
+
+#### Extract Container Images from Catalogs
+
+We propose to extract and collect all the container images used in the Tekton resource (`task` or `pipeline`) by iterating through all the values in the `tasks.steps.image` fields of the resource, for example, `bash:latest` and `alpine` are extracted in the following `pipeline`:
+
+```yaml
+apiVersion: tekton.dev/v1beta1
+kind: Pipeline
+metadata:
+  name: demo-pipeline
+...
```

Comment:
should we use Pipeline as an example here? Asking as we haven't laid out guidelines as how pipelines should be organised within the catalog. This might create confusion. @jerop @vdemeester please let me know your thoughts.

Reply:
I tend to agree with @vinamra28 on this. I feel we should give the example on the `Task` to start with 👼🏼

Reply:
Thanks for the feedback @vinamra28 and @vdemeester! I will use `Task` as the first example

I agree that pipeline is not the main focus of Catalog, but it is already surfaced as a "kind" in Artifact Hub, so I think it still worth adding a pipeline example and callout what tasks in the pipeline will/ won't be scanned